blocks/tabs/tabs.css

```diff
@@ -0,0 +1,278 @@
+  }
+
+  100% {
+      opacity: 1;
+      transform: translate3d(0, 0, 0);
+  }
+}
+
+.tabs {
+  -webkit-font-smoothing: antialiased;
+
+  .tab-component {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    width: 100vw;
+
+    p:last-child {
+      margin-bottom: 0;
+      display: flex;
+      justify-content: center;
+
+      picture {
+        display: block;
+        position: relative;
+        width: 90vw;
+        aspect-ratio: 1/1;
+      }
+
+      img {
+        position: absolute;
+        bottom: 0;
+        width: 100%;
+        height: 100%;
+      }
+    }
+
+    .tab-nav {
+      position: relative;
+      list-style-type: none;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      width: 96vw;
+      font-family: var(--calcite-sans-family);
+      padding: 0;
+      margin: 0 auto;
+
+      calcite-icon {
+        color: var(--calcite-ui-inverse);
+      }
+
+      .arrow-button {
+        position: absolute;
+        border: none;
+        padding: 0;
+        width: 40px;
+        height: 40px;
+        background-color: var(--calcite-ui-foreground-2); 
+        opacity: .96;
+        margin: auto 0;
+      }
+
+      .arrow-button.left {
+        left: 0;
+      }
+
+      .arrow-button.right {
+        right: 0;
+      }
+
+      .tab-title {
+        display: none;
+        border-bottom: 1px solid var(--calcite-ui-border-2);
+        padding-bottom: 2px;
+
+        button {
+          background: none;
+          color: white;
+          font-size: var(--font-0);
+          font-weight: 400;
+        }
+      }
+
+      .tab-title.visible {
+        display: block;
+      }
+
+      .tab-title.selected {
+        border-bottom: 3px solid #0079c1;
+        padding-bottom: 0;
+      }
+    }
+
+    .tab-content {
+      display: none;
+      width: 100vw;
+      position: relative;
+      padding: 0;
+    }
+
+    .tab-content.selected {
+      display: block;
+    }
+
+    p:first-child img {
+      position: absolute;
+      object-fit: cover;
+      top: 0;
+      width: 100%;
+      height: 100%;
+      z-index: -1;
+    }
+
+    .buttons-wrapper {
+      width: 90%;
+      margin: auto;
+      margin-top: 2rem;
+
+      calcite-button {
+        padding-bottom: 0.75rem;
+        margin-bottom: 0.5rem;
+        margin-right: 0.7rem;
+
+        span {
+          color: white;
+        }
+      }
+    }
+
+    .text-wrapper {
+      padding: 0 5vw;
+      padding-top: 4rem;
+
+      p, h2 {
+        width: 90%;
+        font: 400 var(--font-0) / 1.5 var(--calcite-sans-family);
+      }
+
+      p:first-child {
+        font-size: var(--font-0);
+        font-weight: 600;
+        text-transform: uppercase;
+        color: var(--calcite-ui-text-2);
+        margin: 0;
+        margin-bottom: 0.5rem;
+      }
+
+      p:last-child {
+        color: white;
+        margin: 0;
+        margin-bottom: 1.5rem;
+      }
+
+      h2 {
```

Comment:
can you check whether h2 styles are similar across the page? If so, we should probably move this code to styles.css and just overwrite it here selectively.